AppAuth-JS/app/redirect.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
```

Comment:
add license header

Reply:
this is Google's lib fork. Seems Google has missed license, and I did not edit this file